docs/sections/cn_workflows.md

```diff
@@ -211,7 +211,26 @@ popper run -f /home/user/proj/wf.yml -w /home/user/proj/
 
 The above writes the `/home/user/proj/myfile` even though Popper is 
 being invoked from `/tmp`, since the `-w` flag is being passed to 
-`ppopper run`.
+`popper run`.
+
+### Changing the working directory
+
+To specify a working directory for a step you can use the `dir` attribute
+in the workflow. This will only going to change where the specified
+command is executed.
+
+For example, adding `dir` to a workflow would looks like the following:
```

Comment:
this can be ".. to a workflow would look like the following:" or "to a workflow results in the following:"